dev-libs/xxhash/xxhash-0.6.4.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A fast non-cryptographic hash algorithm"
+DESCRIPTION="xxHash is an extremely fast non-cryptographic hash algorithm, working at speeds close to RAM limits."
+HOMEPAGE="http://cyan4973.github.io/xxHash/"
+COPYRIGHT="2012-2016 Yann Collet"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://github.com/Cyan4973/xxHash/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="4570ccd111df6b6386502791397906bf69b7371eb209af7d41debc2f074cdb22"
+SOURCE_DIR="xxHash-$portVersion"
```

Comment:
Our [guidelines][0] requires that archives must be in `<portName>-<portVersion>` format.

```diff
 CHECKSUM_SHA256="4570ccd111df6b6386502791397906bf69b7371eb209af7d41debc2f074cdb22"
+SOURCE_FILENAME="xxhash-$portVersion.tar.gz"
 SOURCE_DIR="xxHash-$portVersion"
```
[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name